cloudsplaining/command/create_exclusions_file.py

```diff
@@ -29,12 +28,18 @@
     required=True,
     help="Relative path to output file where we want to store the exclusions template.",
 )
-@click_log.simple_verbosity_option(logger)
-def create_exclusions_file(output_file):
+@click.option(
+    '--verbose','-v',
+    type=click.Choice(['critical', 'error', 'warning', 'info', 'debug'],
+    case_sensitive=False))
+def create_exclusions_file(output_file, verbose):
     """
     Creates a YML file to be used as a custom exclusions template,
     so users can fill out the fields without needing to look up the required format.
     """
+    if verbose:
+        log_level = getattr(logging, verbose.upper())
+        change_log_level(log_level)
```

Comment:
change the log_level of the "cloudspanning" logger which is a parent to all submodule loggers